workshop-registration.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title>WebDev Workshop Registration</title>
+    </head>
+    <body>
+        <h1 id="main-title">WebDev Workshop Registration</h1>
+        <p id="desc1">
+        Join our immersive workshop to sharpen your modern web development skills.<br>
+        Open to all, from beginners to experts—select sessions that match your passion and goals!
+        </p>
+        <form action="https://formspree.io/f/your-form-id" method="POST">
```

Comment:
The `action` value is wrong. You need to copy the exact URL from inside your formspree account

<img width="1685" height="1397" alt="CleanShot 2025-07-23 at 14 20 41" src="https://github.com/user-attachments/assets/1d81053b-6738-4c87-bdbc-829c2f77516f" />